R/blueprint.R

```diff
@@ -26,6 +26,9 @@
 #' prediction time? This information is used by the `clean` function in the
 #' `forge` function list, and is passed on to [scream()].
 #'
+#' @param composition Either "tibble", "matrix", or "dgCMatrix" for the format
+#' of the processed predictors.
```

Comment:
```
Either `"tibble"`, `"matrix"`, or `"dgCMatrix"` for the format
of the processed predictors. If `"matrix"` or `"dgCMatrix"` are chosen,
all of the predictors must be numeric after the preprocessing method
has been applied, otherwise an error is thrown.
```

I don't think we have mentioned the numeric restriction anywhere, and this seemed like a good place

Reply:
Maybe also mention that if dgCMatrix is used, the Matrix package must be installed

Reply:
Matrix is a "recommended" package, which I believe means it is part of regular R installations, right? It says [that here FWIW](http://matrix.r-forge.r-project.org/).

Reply:
I did add `rlang::if_installed` to send an informative error in case someone happens to not have this installed but I'm not sure how that would typically happen.